src/block/progress-circle/style.scss

```diff
@@ -0,0 +1,72 @@
+@import "common";
+
+.stk-block-progress-circle {
+	display: flex;
+}
```

Comment:
Spaces and tabs seem to be mixed up here, you may need to install **Stylelint** in your VS Code to lint this css file properly.